build_stats.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh
+
+set -e
+
+HERE_DIR="$(cd "$(dirname "$1")"; pwd)/$(basename "$1")"
```

Comment:
OK, let's just bite the bullet and do:

```sh
HERE_DIR=$(dirname $(readlink -f "$0"))
```

and if they don't have `readlink` they should install it. My guess is most proper Unix people do (on Linux and OS X it's probably under the `core-utils` package).